plugins/inventory/scaleway.py

```diff
@@ -31,6 +31,9 @@
         tags:
             description: Filter results on a specific tag.
             type: list
+        scw_profile:
+            description: The config profile to use in config file
```

Comment:
```suggestion
            description:
            - The config profile to use in config file.
            - By default uses the one specified as C(active_profile) in the config file, or falls back to C(default) if that is not defined.
```

Reply:
Thanks for review, i added it in last commit